README.md

```diff
@@ -338,7 +338,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Cynthia Teeters](https://github.com/cynthiateeters)
 
 - ### **D**
-
+  - [David Okononfua](https://github.com/Vinyl-Davyl)
```

Comment:
Please add your name in line 355, after David Leal. Let us know if you need any help.

Reply:
Alright,
I guess its an alphabetical allignment!